Resources/Prototypes/Recipes/Lathes/cooking.yml

```diff
@@ -4,36 +4,38 @@
   result: ButchCleaver
   completetime: 500
   materials:
-    steel: 300
+    Steel: 300
 
 - type: latheRecipe
   id: KitchenKnife
-  icon: Objects/Weapons/Melee/kitchen_knife.rsi
-  result: KitchenKnife
-  completetime: 400
+  icon:
+    sprite: Objects/Weapons/Melee/Kitchen_knife.rsi
```

Comment:
should be kitchen_knife.rsi, no?

Reply:
is Kitchen_knife.rsi It works like that so yeah

Reply:
This works for some reason but it really shouldn't.

Reply:
why shouldn't it work thats the name of the file?